pygenometracks/utilities.py

```diff
@@ -261,7 +261,7 @@ def transform(score_list, transform, log_pseudocount, file):
         else:
             return(np.log1p(score_list))
     elif transform == '-log':
-        if np.nanmax(score_list) <= - log_pseudocount:
```

Comment:
why did you change the max to min?

Reply:
this should stay max, i would say. it is enough only if one is below this threshold.

Reply:
No if one is below but not others, the max will be above.

Reply:
Now I am confused... I thought we don't want to apply log to negative values. So if one value is below we don't want to compute, so if the min is below, no?

Reply:
I understood that at the moment we cannot accept any negative value so you are giht and we can change that to min.